src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ApartmentValueFunction.java

```diff
@@ -23,158 +23,134 @@
 public class ApartmentValueFunction {
 
 	/**
-	 * @floorAreaValueFunction A {@link PartialValueFunction} object on which the
-	 *                         calculation of the floor area subjective values are
-	 *                         based.
+	 *  A {@link PartialValueFunction} object on which the calculation of the floor area subjective values 
```

Comment:
Évitez tous ces liens inutiles distrayants SVP. (OPT)

Reply:
Fixed in commit 74bf0331b6e997409de3a30a47a03d5ca6561aa1